docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+
+## Verifying events were received 
+
+After sending events, verify that the events were received by the appropriate `Subscribers`.
+
+1. Look at the logs for the `hello-display` event consumer with the following command:
+
+```sh
+kubectl -n event-example logs -l app=hello-display
+```
+
+
+The command shows the `Attributes` and `Data` of the events you sent to `hello-display`:
+
+```sh
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 0.2
+  type: greeting
+  source: not-sendoff
+  id: say-hello
+  time: 2019-05-20T17:59:43.81718488Z
+  contenttype: application/json
+Extensions,
+  knativehistory: default-broker-srk54-channel-24gls.event-example.svc.cluster.local
+Data,
+  {
+    "msg": "Hello Knative!"
+  }
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 0.2
+  type: greeting
+  source: sendoff
+  id: say-hello-goodbye
+  time: 2019-05-20T17:59:54.211866425Z
+  contenttype: application/json
+Extensions,
+  knativehistory: default-broker-srk54-channel-24gls.event-example.svc.cluster.local
+Data,
+  {
+    "msg": "Hello Knative! Goodbye Knative!"
+  }
+```
+
+2. Look at the logs for the `goodbye-display` event consumer with the following command:
+
+```sh
+kubectl -n event-example logs -l app=goodbye-display
+```
+
+The command shows the `Attributes` and `Data` of the events you sent to `goodbye-display`:
+
+
+```sh
+☁️  cloudevents.Event
+ Validation: valid
+ Context Attributes,
+   specversion: 0.2
+   type: not-greeting
+   source: sendoff
+   id: say-goodbye
+   time: 2019-05-20T17:59:49.044926148Z
+   contenttype: application/json
+ Extensions,
+   knativehistory: default-broker-srk54-channel-24gls.event-example.svc.cluster.local
+ Data,
+   {
+     "msg": "Goodbye Knative!"
+   }
+ ☁️  cloudevents.Event
+ Validation: valid
+ Context Attributes,
+   specversion: 0.2
+   type: greeting
+   source: sendoff
+   id: say-hello-goodbye
+   time: 2019-05-20T17:59:54.211866425Z
+   contenttype: application/json
+ Extensions,
+   knativehistory: default-broker-srk54-channel-24gls.event-example.svc.cluster.local
+ Data,
+   {
+     "msg": "Hello Knative! Goodbye Knative!"
+   } 
+```
+
+These results should match up with the events you created in **Sending CloudEvents to the Broker**. If they do not, see the [Debugging Guide](./debugging/README.md) for more information.
+
+## Cleaning up
+
+Delete the namespace to conserve resources:
+
+```sh
+kubectl delete namespace event-example
+```
+
+## What’s next 
+
+You've learned the basics of the Knative Eventing workflow. Here are some additional resources to help you continue to build with the Knative Eventing component.
+
+- [Knative Eventing Overview](./README.md)
```

Comment:
As per the [draft documentation](https://docs.google.com/document/d/1nxNNzHoVoWulibSZBFGHo8EjVk47FK55NtfcBDto8w0/), the plan was to have a Getting started section with following topics:
1. Quick Start: Deploy "Hello World" eventing solution
2. Step-by-step: Deploy "Hello World" eventing solution --> This is the current PR
3. Using event importer to consume events
4. Configure Broker to use a different Channel

Question: Above 4 topics have a gradual flow (continuation) from #1 to #4. Is there a plan to maintain that flow and create more pages and have pre-req and next steps?

Reply:
@akashrv  I'd think of this PR as 1 + 2 merged together. We are not currently planning to create the stripped-down, no explanation version of this guide that does all this work in one step. 

For #3, I am beginning to work on that. I've got a meeting with Ville tomorrow.

#4 is further out at this point; I don't have a timeline for when that will be covered.

When #3 exists, it will definitely be linked to from this guide as a good next step, and same for #4 further down the road.